packages/shared/trade/src/borrow-terms/use-borrow-terms.tsx

```diff
@@ -35,7 +35,11 @@ export const useBorrowTerms = (
   const nonLeveragedYields = currentNetworkStore.getAllNonLeveragedYields();
 
   const spotMaturityData = useSpotMaturityData(
-    deposit ? availableDebtTokens : []
+    deposit
```

Comment:
Does this selector get the right type?
![image](https://github.com/user-attachments/assets/2b2b25e8-dbf0-49af-8817-6617f70e3f43)

Reply:
deposit comes through as  TokenDefinition | undefined. This is a only used on the leveraged pages that are known to be broken at this time.